### PR TITLE
feat(form/inputs): add support for image drop+paste in PTE input

### PR DESCRIPTION
**Pull request by sjelfull**
### Description

This adds support for drag-n-drop + pasting of images and files into the Portable Text Input inside the studio.

### Testing

1. Open [this test document](https://test-studio-git-feat-edx-1176-image-clipboard-support.sanity.build/test/structure/input-standard;portable-text;pt_allTheBellsAndWhistles;757b15ed-d091-464d-9422-577956b5d293) in the test studio
2. Try to test the different scenarios on the Text field:
  - Drag 1 image
  - Drag 2 images
  - Drag a invalid file (something not PDF)
  - Drag 1 PDF file
  - Drag 2 PDF files
3. Now do the same, but pasting instead.

### Notes about PR
- Paste component test is skipped in Firefox because the security model doesn't allow us to make a synthetic test

### Notes for release

- The Portable Text Input now supports drag-n-dropping + pasting of images and files into `Image` and `File` blocks

### CODE REVIEW

File: dev/test-studio/schema/standard/portableText/allTheBellsAndWhistles.ts [hunk 1]
```diff
@@ -13,6 +13,9 @@ export const ptAllTheBellsAndWhistlesType = defineType({
     {
       name: 'whitespace',
       title: 'Whitespace',
+      options: {
```
Comment (jtpetty) [inline dev/test-studio/schema/standard/portableText/allTheBellsAndWhistles.ts:16]: is this needed for this change?

Comment (sjelfull) [reply dev/test-studio/schema/standard/portableText/allTheBellsAndWhistles.ts]: Nope, removed

File: packages/@sanity/portable-text-editor/src/editor/plugins/createWithInsertData.ts [hunk 2]
```diff
@@ -169,7 +169,16 @@ export function createWithInsertData(
       change$.next({type: 'loading', isLoading: true}) // This could potentially take some time
       const html = data.getData('text/html')
       const text = data.getData('text/plain')
-      if (html || text) {
+
+      const {files} = data
```
[the remark this answers is not in the document]
Comment (sjelfull) [reply packages/@sanity/portable-text-editor/src/editor/plugins/createWithInsertData.ts:173]: Added component tests for this now

**Review by jtpetty — APPROVED** [review]
👍 - thank you for adding tests!

**Pull request merged**